js/controllers/form/DOIPubIdExportFormHandler.js

```diff
@@ -25,10 +25,34 @@
 			function($formElement, options) {
 		this.parent($formElement, options);
 
-		$('#exportSubmissionXmlForm a[id*="-button-"]').click(this.callbackWrapper(
-				this.submitAction_));
-		$('#exportIssueXmlForm a[id*="-button-"]').click(this.callbackWrapper(
-				this.submitAction_));
+		this.submissionsActions_ = [];
+		this.issuesActions_ = [];
+		this.actionRegExp_ = [];
+
+		var i;
+		if (options.submissionsActions) {
+			for (i = 0; i < options.submissionsActions.length; i++) {
+				/** @type {{_id:string}}*/
+				this.optionsSubmissionsAction = options.submissionsActions[i];
+				this.submissionsActions_.push(this.optionsSubmissionsAction._id);
```

Comment:
Hmm, _id comes from the PHP object via `json_encode`, correct? I'd rather not access a private/protected property directly.